version/version.go

```diff
@@ -12,7 +12,7 @@ var (
 	// VersionPrerelease is A pre-release marker for the Version. If this is ""
 	// (empty string) then it means that it is a final release. Otherwise, this
 	// is a pre-release such as "dev" (in development), "beta", "rc1", etc.
-	VersionPrerelease = "dev"
+	VersionPrerelease = ""
```

Comment:
I would suggest updating the README for the plugin so the exerpt for `required_plugins` shows the latest version as 1.1.2, but this is optional.

Reply:
Ah good point. I will push that and merge. Thanks. I'll do this for the other two plugins as well.

Reply:
Actually it looks like they may be out of sync a bit. I will fast follow with a different PR since the integration install docs use `~> 1` for the version string.